theano/tests/breakpoint.py

```diff
@@ -12,8 +12,8 @@ class PdbBreakpoint(Op):
     conditional breakpoint, inside a theano function, based on a symbolic
     scalar condition.
 
-    @type name: String
-    @param name: name of the conditional breakpoint. To be printed when the
+    :type name: String
+    :param name: name of the conditional breakpoint. To be printed when the
```

Comment:
Yes, numpydoc everywhere is the goal. It does not mean that every time we touch a docstring, we have to make it numpydoc-compatible for the PR to be merged.
This formatting change does not reach the goal (numpydoc) but is strictly better than not changing it (now it displays correctly, before it did not), which is why I merged.